### ISSUE TITLE
google font request under https protocol returns 403 code

### ISSUE THREAD

**Issue by invkrh** [theme-next/hexo-theme-next#613]
### I agree and want to create new issue 

- [X] Yes, I was on [Hexo Docs page](https://hexo.io/docs/), especially on [Templates](https://hexo.io/docs/templates.html), [Variables](https://hexo.io/docs/variables.html), [Helpers](https://hexo.io/docs/helpers.html) and [Troubleshooting](https://hexo.io/docs/troubleshooting.html).
- [X] Yes, I was on [NexT Documentation Site](http://theme-next.org/docs/).
- [X] And yes, I already searched for current [issues](https://github.com/theme-next/hexo-theme-next/issues?utf8=%E2%9C%93&q=is%3Aissue) and this is not help to me.

***

### Expected behavior 
Font should be correctly applied no matter if it is the first visit to the site (cache does not exist)

### Actual behavior 
If the browser did not cache the fonts before, the font cannot be applied because the request to font.googleapi.com returns 403 code.

如果是第一次访问网站，google font 返回403，字体都没有正确显示。但是font.googleapi.com的请求链接可以单独点击，返回200，chrome会cache到本地disk，之后字体才会正常显示。
试过在单独的网页上调用同样的request，字体显示均正常。

### Steps to reproduce the behavior 
1. Go to https://invkrh.me (if this is the first time, the fonts will *NOT* be applied)
2. You can manually cache the font by clicking the link used to install the fonts: `https://fonts.googleapis.com/css?family=Roboto|Lato|Merriweather|Parisienne|Consolas`
3. Refresh the blog page, fonts are applied.
4. In chrome, open dev tools -> network -> toggle disable cache
5. Go to https://invkrh.me, since no cache exists, the fonts are *NOT* applied, just like first time visit.

Error message:
```Failed to load resource: the server responded with a status of 403 ()```

* Link to demo site with this issue: N/A
* Link(s) to source code or any usefull link(s): N/A

### Node.js and NPM Information

```
v11.4.0
6.7.0
```

### Package dependencies Information

```
{
  "name": "hexo-site",
  "version": "0.0.0",
  "private": true,
  "hexo": {
    "version": "3.8.0"
  },
  "dependencies": {
    "hexo": "^3.7.0",
    "hexo-browsersync": "^0.3.0",
    "hexo-deployer-git": "^1.0.0",
    "hexo-generator-archive": "^0.1.5",
    "hexo-generator-category": "^0.1.3",
    "hexo-generator-feed": "^1.2.2",
    "hexo-generator-index": "^0.2.1",
    "hexo-generator-searchdb": "^1.0.8",
    "hexo-generator-tag": "^0.2.0",
    "hexo-renderer-ejs": "^0.3.1",
    "hexo-renderer-kramed": "^0.1.4",
    "hexo-renderer-stylus": "^0.3.3",
    "hexo-server": "^0.3.1",
    "hexo-symbols-count-time": "^0.4.4"
  }
}
```

#### Hexo version

```
hexo: 3.8.0
hexo-cli: 1.1.0
os: Darwin 17.7.0 darwin x64
node: 11.4.0
v8: 7.0.276.38-node.13
uv: 1.24.0
zlib: 1.2.11
ares: 1.15.0
modules: 67
nghttp2: 1.34.0
napi: 3
llhttp: 1.0.1
http_parser: 2.8.0
openssl: 1.1.0j
icu: 63.1
unicode: 11.0
cldr: 34.0
tz: 2018e
```

#### Hexo Configuration

```yml
```

### NexT Information

**NexT Version:**

- [ ] Latest Master branch
- [X] Latest Release version
- [ ] Old version

**NexT Scheme:**

- [ ] All schemes
- [ ] Muse
- [X] Mist
- [ ] Pisces
- [ ] Gemini

#### NexT Configuration:

```yml
font:
  enable: true

  # Uri of fonts host. E.g. //fonts.googleapis.com (Default).
  host:

  # Font options:
  # `external: true` will load this font family from `host` above.
  # `family: Times New Roman`. Without any quotes.
  # `size: xx`. Use `px` as unit.

  # Global font settings used for all elements in <body>.
  global: # mono, sans serif
    external: true
    family: Roboto
    # size: 18

  # Font settings for Headlines (H1, H2, H3, H4, H5, H6).
  # Fallback to `global` font settings.
  headings: # mono, sans serif
    external: true
    family: Lato
    size:

  # Font settings for posts.
  # Fallback to `global` font settings.
  posts: # serif
    external: true
    family: Merriweather

  # Font settings for Logo.
  # Fallback to `global` font settings.
  logo: # handwriting
    external: true
    family: Parisienne

  # Font settings for <code> and code blocks.
  codes: # mono, sans serif
    external: true
    family: Consolas
    # size: 18
```

### Other Information 
Fonts are applied if cache exists
![image](https://user-images.githubusercontent.com/5421623/53132508-30a15700-3570-11e9-8425-a7132eade63d.png)

If cache does not exist (first time visit), fonts are not applied since font request failed with 403 code
![image](https://user-images.githubusercontent.com/5421623/53132750-f6848500-3570-11e9-9b66-c30eb9051b80.png)

**Comment by stevenjoezhang** [comment]
Seems https://fonts.googleapis.com will return 403 if the request headers include `referer: https://invkrh.me/`
![2019-02-21 10 41 02](https://user-images.githubusercontent.com/16272760/53139680-66bcf600-35c5-11e9-8b3b-ba38e44bb758.png)
![2019-02-21 10 41 14](https://user-images.githubusercontent.com/16272760/53139682-6886b980-35c5-11e9-9b90-1083541d6036.png)
you can try to use a mirror site of google font, e.g. fonts.loli.net, instead of fonts.googleapis.com

**Comment by invkrh** [comment]
@stevenjoezhang Thank you for your help. Nice catch!
I add a `referrerpolicy="no-referrer"` to the font `<link>` and it works.

**Comment by sli1989** [comment]
`fonts.loli.net` works fine? any other recommendation? @stevenjoezhang

**Comment by stevenjoezhang** [comment]
@sli1989 Yes, there are many mirror sites of `fonts.googleapis.com`:
`fonts.loli.net`
`fonts.googleapis.cnpmjs.org`
`fonts.proxy.ustclug.org`

**Comment by sli1989** [comment]
for me, `fonts.loli.net` is worst than `fonts.googleapis.com`. 😢 
I close the google font setting...

**Comment by stevenjoezhang** [comment]
@sli1989 Seems fonts.loli.net is just a proxy... Maybe it's blocked by the wall during the Plenary meeting of the National People's Congress...
